README.rdoc

```diff
@@ -349,7 +349,7 @@ You need exiftool on your system in order to successfully run all of the tests.
 
 To run all tests, use the command below.  Note that to run integration tests, you will need to be connected to the VPN (even if on campus).
 
-    bin/run_all_tests
+    bundle exec rake run_all_tests
```

Comment:
I don't understand what this has to do with honeybadger.  Should this be in a different PR?